CMakeLists.txt

```diff
@@ -440,6 +435,7 @@ option(WITH_RADOSGW "Rados Gateway is enabled" ON)
 option(WITH_RADOSGW_FCGI_FRONTEND "Rados Gateway's FCGI frontend is enabled" OFF)
 option(WITH_RADOSGW_BEAST_FRONTEND "Rados Gateway's Beast frontend is enabled" ON)
 option(WITH_RADOSGW_BEAST_OPENSSL "Rados Gateway's Beast frontend uses OpenSSL" ON)
+option(WITH_RADOSGW_AMQP_ENDPOINT "Rados Gateway's pubsub support for AMQP push endpoint" ON)
```

Comment:
not very familiar with cmake, tbh, but is this checking the same things as the code you removed above? I see the above code checking for the header, but this seems to only define the option, without specifying the required header.

Reply:
well, disregard. That's happening below, in the `FindRabbitMQ.cmake` file.